test/Spaces/sphere.jl

```diff
@@ -52,6 +52,59 @@ using Test
     end
 end
 
+@testset "Bubble correction Nq robustness" begin
+
+    for FT in (Float64, Float32)
+        no_bubble_rtols = (
+            FT(0.64),
+            FT(0.19),
+            FT(0.027),
+            FT(0.0049),
+            FT(0.0008),
+            FT(0.00014),
+            FT(2.4e-5),
+            FT(3.97e-6),
+            FT(6.77e-7),
+        )
```

Comment:
Can we replace this by something of the form `tol = a * 0.5 ^ 2Nq`?

Reply:
It does not appear to be quadratic actually. More linear 

```julia
order = log(abs( (no_bubble_rtols[9] - no_bubble_rtols[8]) / (no_bubble_rtols[8] - no_bubble_rtols[7]) ) ) / log(abs((no_bubble_rtols[8] - no_bubble_rtols[7])/(no_bubble_rtols[7] - no_bubble_rtols[6])))

1.027940246667871
```

Reply:
There is no way we can capture a simple rule for all tols in a way that we can have them `broken` for `Nq=2` and `Nq` odd and pass otherwise. I tried.